__docs__/wonder-blocks-link/link.stories.tsx

```diff
@@ -83,10 +83,8 @@ Primary.play = async ({canvasElement}) => {
 
     // Focus style with keyboard navigation
     await userEvent.tab();
-    const computedStyle = getComputedStyle(link, ":focus-visible");
-    // rgb(24, 101, 242) is the same as Color.blue. `toBe` doesn't seem to
-    // compare different color formats, so hex was converted to RGB.
-    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 1px");
+    // rgb(24, 101, 242) is the same as Color.blue
+    await expect(link).toHaveStyle("outline: rgb(24, 101, 242) solid 1px");
```

Comment:
**note:** This and other similar assertions where changed b/c `getComputedStyle` doesn't work with the new testing approach.

I got these errors the first time I integrated the new SB test CLI in CI, and made these changes to fix the failures.

Reply:
What do we think about tests that check for specific styles? I had started removing jest tests that were asserting styles since it would be visually covered by Chromatic (especially with the "All Variants" stories + pseudostates add on!). Is it helpful to also have these component test checks for styles?  

Kinda related - In general, when would it be helpful for us to reach for component tests in Storybook? I'm thinking if there's functionality we want to test for that relies on browser behaviour! Curious to see what other think!

Reply:
100% agree with you. We should move this to "All variants" but the issue is that `Link` (and `Button`) still rely on `Clickable` for setting pseudo states styles. As you know, I'd love to get us to a point where we don't use Clickable for those styles, and we let CSS handle that for us, but that's a tricky change that would be definitively nice to address.

Once we migrate to CSS pseudoclasses, I'm totally in favor of moving this to visual regression tests instead of interaction tests.